### PR TITLE
Substitute occurrences of master by main

### PR DESCRIPTION
**Pull request by martamatos**

### CODE REVIEW

File: week2/exercise-template/src/backend/data/movies.json [hunk 1]
```diff
@@ -1328,7 +1328,7 @@
         "id": 166
     },
     {
-        "title": "A Nightmare on Elm Street 4: The Dream Master",
+        "title": "A Nightmare on Elm Street 4: The Dream main",
```
Comment (dpfernandes) [inline week2/exercise-template/src/backend/data/movies.json]: I think this is a movie title :) 

Should we replace the whole title @martamatos ?

File: week2/exercise-template/src/backend/data/movies.json [hunk 2]
```diff
@@ -24600,23 +24600,23 @@
...
+        "title": "main and Commander: The Far Side of the World",
         "year": 2003,
         "rating": 7.4,
         "votes": 179736,
         "running_times": 8280,
         "id": 3076
     },
     {
-        "title": "Masterminds",
+        "title": "mainminds",
```
Comment (dpfernandes) [inline week2/exercise-template/src/backend/data/movies.json]: The same here

File: week2/exercise-template/src/backend/data/movies.json [hunk 3]
```diff
@@ -24600,23 +24600,23 @@
...
         "year": 2016,
         "rating": 5.8,
         "votes": 36500,
         "running_times": 5700,
         "id": 3077
     },
     {
-        "title": "Masters of the Universe",
+        "title": "mains of the Universe",
```
Comment (dpfernandes) [inline week2/exercise-template/src/backend/data/movies.json]: the same here

File: week2/exercise-template/src/backend/data/movies.json [hunk 4]
```diff
@@ -38128,7 +38128,7 @@
         "id": 4766
     },
     {
-        "title": "The Beastmaster",
+        "title": "The Beastmain",
```
Comment (dpfernandes) [inline week2/exercise-template/src/backend/data/movies.json]: the same here

File: week2/exercise-template/src/backend/data/movies.json [hunk 5]
```diff
@@ -43560,15 +43560,15 @@
         "id": 5445
     },
     {
-        "title": "The Master",
+        "title": "The main",
```
Comment (dpfernandes) [inline week2/exercise-template/src/backend/data/movies.json]: the same here

File: week2/exercise-template/src/backend/data/movies.json [hunk 6]
```diff
@@ -43560,15 +43560,15 @@
...
         "year": 2012,
         "rating": 7.1,
         "votes": 117674,
         "running_times": 8280,
         "id": 5446
     },
     {
-        "title": "The Master of Disguise",
+        "title": "The main of Disguise",
```
Comment (dpfernandes) [inline week2/exercise-template/src/backend/data/movies.json]: the same here

File: week2/exercise-template/src/backend/data/movies.json [hunk 7]
```diff
@@ -44384,7 +44384,7 @@
         "id": 5548
     },
     {
-        "title": "The Pagemaster",
+        "title": "The Pagemain",
```
Comment (dpfernandes) [inline week2/exercise-template/src/backend/data/movies.json]: the same here

File: week2/exercise-template/src/backend/data/movies.json [hunk 8]
```diff
@@ -51272,7 +51272,7 @@
         "id": 6409
     },
     {
-        "title": "Wishmaster",
+        "title": "Wishmain",
```
Comment (dpfernandes) [inline week2/exercise-template/src/backend/data/movies.json]: the same here

**Review by dpfernandes — CHANGES_REQUESTED** [review]

File: week2/exercise-template/src/backend/data/movies.json (hunk 7 above)
Comment (CK-HYF-DK) [reply week2/exercise-template/src/backend/data/movies.json]: this one is kinda funny

File: week2/exercise-template/src/backend/data/movies.json (hunk 5 above)
Comment (CK-HYF-DK) [reply week2/exercise-template/src/backend/data/movies.json]: Another Paul Thomas Anderson classic :D

**Pull request merged**